ext/hal/atmel/asf/common/components/wifi/winc1500/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+zephyr_include_directories(
+  ./
+  $ENV{ZEPHYR_BASE}/include/drivers
+  $ENV{ZEPHYR_BASE}/drivers/wifi
```

Comment:
This line is redundant with something done in drivers/wifi/CMakeLists.txt and can be removed.